app/design/frontend/Magebit/Learning/Magento_Catalog/templates/product/view/attributes.phtml

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * @copyright Copyright (c) 2024 Magebit (https://magebit.com)
+ * @author    Magebit <info@magebit.com>
+ * @license   GNU General Public License ("GPL") v3.0
+ */
+/** @var \Magento\Catalog\Block\Product\View\Attributes $block */
+/** @var \Magento\Catalog\Model\Product $product */
+$product = $block->getProduct();
+$attributes = $product->getAttributes();
+$catalogHelper = $this->helper('Magento\Catalog\Helper\Output');
+// Attributes to display
+$displayAttributes = ['dimensions', 'color', 'material'];
```

Comment:
better to create your own block class and define these in block

Reply:
Not resolved yet.
Create your own block to add to the functionality and use it here